internal/data/payments.go

```diff
@@ -264,6 +264,39 @@ func (p *PaymentModel) GetAll(ctx context.Context, queryParams *QueryParams, sql
 	return payments, nil
 }
 
+func (p *PaymentModel) GetAllReadyToPatchAnchorTransactions(ctx context.Context) ([]Payment, error) {
+	const query = `
+		SELECT
+			p.id,
+			p.status,
+			rw.id AS "receiver_wallet.id",
+			rw.anchor_platform_transaction_id AS "receiver_wallet.anchor_platform_transaction_id"
```

Comment:
👀 Populating only a few fields in an object is not a good practice, because when you're manipulating an object, how would you know that most fields are empty and that you can't rely on the info of all these fields?

Maybe you can grab only the IDs and populate the whole objects using `GetByIDs`? Or use a reusable method to build the queries related to Payments? This could be done in another PR though, since this looks like a big change.

I encountered this same problem in a few other data models, such as ReceiverWallets: the queries are repeated all over, and adding new fields has become a hassle.

Reply:
I agree with moving this to another refactor PR.

Reply:
Cool, can you plz create a ticket for that refactor?